### PR TITLE
Add position utilities class

### LINKED ISSUE #18476: Add position utility classes
Relative position class could be a real time saver in many situations.
`.pos-r{
    position: relative
}`

### PR DESCRIPTION
**Pull request by CVarisco**
Utility class to set `position` property. Old PR #19292 with discussion. Fixes #18476.

### CODE REVIEW

File: scss/utilities/_position.scss [hunk 1]
```diff
@@ -23,3 +23,8 @@
     z-index: $zindex-sticky;
   }
 }
+
+.pos-s { position: static; }
```
Comment (mdo) [inline scss/utilities/_position.scss]: You're missing `position: sticky`, which, with this naming scheme, would be problematic. We'll need that utility and a different name.

**Review by mdo — CHANGES_REQUESTED** [review]

File: scss/utilities/_position.scss (hunk 1 above)
Comment (CVarisco) [inline scss/utilities/_position.scss]: You are right!
Could be a good naming scheme ?
`pos-sti`
`pos-sta`
`pos-rel`
`pos-abs`
`pos-fix`

Comment (mdo) [inline scss/utilities/_position.scss]: At that point, we might as well use the full value names. `.p-sticky`, `.p-static`, etc.

Comment (CVarisco) [inline scss/utilities/_position.scss]: Cool!
Thank you, I will a commit with this edit 🙂

Comment (jipexu) [inline scss/utilities/_position.scss]: why not keep pos- or po- as prefix as p is use for padding ?

Comment (mdo) [inline scss/utilities/_position.scss]: Because there's no risk of overlap with padding—these values are unique to positioning.

**Comment by mdo** [comment]
I think your rebase went awry a bit :).

**Comment by CVarisco** [comment]
ops, I'm sorry for that 🙁 
would you like that I squash commits ?

**Comment by mdo** [comment]
I can squash as I merge when it comes to that :)

**Comment by mdo** [comment]
Closing as this history got messed up.

**Pull request closed without merging**